plugins/Feedback/Controller.php

```diff
@@ -37,12 +35,32 @@ function index()
     public function updateFeedbackReminderDate()
     {
         Piwik::checkUserIsNotAnonymous();
+
+        $nextReminder = Common::getRequestVar('nextReminder');
+
+        if ($nextReminder !== Feedback::NEVER_REMIND_ME_AGAIN) {
+            $nextReminder = Date::now()->getStartOfDay()->addDay($nextReminder)->toString('Y-m-d');
+        }
+
+        $feedbackReminder = new FeedbackReminder();
+        $feedbackReminder->setUserOption($nextReminder);
+
+        return json_encode(['Next reminder date: ' . $nextReminder]);
```

Comment:
You might need to also call `Piwik\DataTable\Renderer\Json::sendHeaderJSON()` here and below because I don't think controller methods do that by default.